### ISSUE TITLE
 support custom copy icon

### ISSUE THREAD

**Issue by xiaoxinbo118** [mac-s-g/react-json-view#345]
I want to relace copy icon。Would do you support custom copy icon and other icons?

Thanks.

**Comment by 55Cancri** [comment]
Yes, please allow custom copy, delete, and add icons such as font awesome, and also allow custom positioning. For example, all icons display on the far right of the json as you hover over different rows.
